generator/sources/bpmn/build.xml

```diff
@@ -36,43 +36,36 @@
 			</classpath>
 		</taskdef>
 	</target>
+	
+	<!-- replace with an task - replace -->
+	<target name="replace-tokens">
```

Comment:
I would check if ${basedir}/deploy/src/activiti.cfg.xml is not already exists first.